src/headless/celery_app.py

```diff
@@ -0,0 +1,24 @@
+# coding=utf-8
+from celery import Celery
+
+__author__ = 'Rizky Maulana Nugraha <lana.pcfre@gmail.com>'
+__date__ = '1/28/16'
+
+
+app = Celery('headless.tasks')
+app.config_from_object('headless.celeryconfig')
+
+packages = (
+    'headless',
+)
+
+
+# initialize qgis_app
+from safe.test.utilities import get_qgis_app  # noqa
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
```

Comment:
Could you try the one that @Gustry recommends?

```
from safe.test.qgis_app import qgis_app
APP, IFACE = qgis_app()
```

Reply:
I tried that, but I got
```
OK
make: [coverage-tests] Error 139 (ignored)
```

I will use the old one, since there is no Error comes up.

Reply:
Is-it the only error you have?

Reply:
Yes, not sure what is it about.